src/book/01-team/01-introduction/03-current-team.mdx

```diff
@@ -48,47 +46,37 @@ These members have been on the team for a number of years and have attended at l
 
 ### Members
 
-These members are no longer new members but not yet senior members. Generally they have been on the team for at least a year and have contributed to the team in some way.
+These members are no longer new members but not yet senior members. Generally they have contributed in some way and have found an area on the team.
+
+- **Lachlan Court**: Undergraduate student studying Computer Science. Vision and Webots.
+
+- **Abigail Hall**: Undergraduate student studying a combined degree in Mathematics and Computer Science. Optimisation and motion.
+
+- **Moosa Hassan**: Undergraduate student studying Software Engineering. Vision.
 
-- **Samuel Dearie**: Undergraduate student studying Software Engineering. Works on adding features to NUbook.
+- **Yosiah de Koeyer**: Undergraduate student studying a combined degree in Computer Engineering and Computer Science. DevOps.
 
-- **Moosa Hassan**: Undergraduate student studying Software Engineering.
+- **Alana Noonan**: Undergraduate student studying Computer Science. Computer graphics - NUpbr and Webots model.
+
+- **Benjamin Young**: Undergraduate student studying Computer Systems Engineering. Hardware.
```

Comment:
@xXBenjaminYoungXx you're in a combined degree now aren't you?

Reply:
This is the only place we call it "Computer Systems Engineering", thats what UON calls it, but everywhere else just calls it CE.

Reply:
Just fixed this up